src/main/java/com/ezylang/evalex/operators/arithmetic/InfixPlusOperator.java

```diff
@@ -40,6 +41,17 @@ public EvaluationValue evaluate(
           leftOperand
               .getNumberValue()
               .add(rightOperand.getNumberValue(), expression.getConfiguration().getMathContext()));
+    } else if (leftOperand.isDateTimeValue() && rightOperand.isDurationValue()) {
+      return new EvaluationValue(
+          leftOperand.getDateTimeValue().plus(rightOperand.getDurationValue()));
+    } else if (leftOperand.isDurationValue() && rightOperand.isDurationValue()) {
+      return new EvaluationValue(
+          leftOperand.getDurationValue().plus(rightOperand.getDurationValue()));
+    } else if (leftOperand.isDateTimeValue() && rightOperand.isNumberValue()) {
+      return new EvaluationValue(
+          leftOperand
+              .getDateTimeValue()
+              .plus(Duration.ofDays(rightOperand.getNumberValue().longValue())));
```

Comment:
Unsure  about if days is here the right choice, some people might expect it to be milliseconds

Reply:
I think it depends on the use case. I agree that without a precise use case we can add milliseconds instead of days by default